Cython/Utility/ImportExport.c

```diff
@@ -336,21 +407,32 @@ static int ${import_star}(PyObject* m) {
     if (__Pyx_import_all_from(locals, m) < 0) goto bad;
     list = PyDict_Items(locals);        if (!list) goto bad;
 
+#if CYTHON_COMPILING_IN_LIMITED_API
+    for(i=0; i<PyList_Size(list); i++) {
+        name = PyTuple_Get_Item(PyList_Get_Item(list, i), 0);
+        item = PyTuple_Get_Item(PyList_Get_Item(list, i), 1);
```

Comment:
These will need some error checking I think

Reply:
I think I've added this correctly now, assuming that all error checking is safe to jump to the `bad` label.